pkgs/servers/monitoring/alerta/client.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "alerta";
+  version = "8.3.0";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "83c7d751bad0cb9bd7886700da4cd83c5451b2e8eb8d4cc697966e02d6a565f8";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    six click requests requests-hawk pytz tabulate
+  ];
+
+  doCheck = false;
+
+  disabled = python3.pkgs.pythonOlder "3.6";
+
+  meta = with lib; {
+    homepage = "https://alerta.io";
+    description = "Alerta Monitoring System command-line interface";
+    license = licenses.asl20;
```

Comment:
Can you add yourself as a maintainer?

Reply:
no

Reply:
I might fix other peoples' packages if I notice they're broken, but I don't want to be responsible for stuff I don't use.

Reply:
If there is no maintainer for the package we should consider removing it.

Reply:
@svanderburg added it.
There's even a NixOS module.